docs/source/best-practices/schema-stitching.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Schema Stitching
```

Comment:
Let's kill this article for now - we don't know what the best practices are here.